sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Helpers/AspNetMvcHelper_IsControllerMethod.cs

```diff
@@ -87,52 +93,58 @@ public class MyController : Controller
             var publicFoo = compilation.GetMethodSymbol("PublicFoo");
             var publicBar = compilation.GetMethodSymbol("PublicBar");
             var publicDiz = compilation.GetMethodSymbol("PublicDiz");
+            var publicNonAction = compilation.GetMethodSymbol("PublicNonAction");
 
             AspNetMvcHelper.IsControllerMethod(publicFoo).Should().Be(true);
             AspNetMvcHelper.IsControllerMethod(publicBar).Should().Be(false);
             AspNetMvcHelper.IsControllerMethod(publicDiz).Should().Be(false);
+            AspNetMvcHelper.IsControllerMethod(publicNonAction).Should().Be(false);
         }
 
         [DataTestMethod]
-        [DataRow("3.0.20105.1")]
+        [DataRow("2.1.3")]
```

Comment:
Please explain why you did this?

Reply:
The test is for ASP.NET MVC Core (the attributes are not present in the desktop version of ASP.NET MVC)